corehq/apps/reports/filters/api.py

```diff
@@ -195,6 +195,43 @@ class MobileWorkersOptionsView(EmwfOptionsView):
     """
     urlname = 'users_select2_options'
 
+    # This endpoint is used by select2 single option filters
+    def post(self, request, domain):
+        self.domain = domain
+        self.q = self.request.POST.get('q', None)
+        try:
+            count, options = self.get_post_options()
+            return self.render_json_response({
+                'items': options,
+                'total': count,
+                'limit': request.POST.get('page_limit', 10),
```

Comment:
Its good to avoid having numbers kept around and rather move them as a meaningful constant like for instance PAGE_LIMIT here. It makes it easier to change it when needed and much easier to figure out related places.